configs/ner_bc5cdr.cfg

```diff
@@ -0,0 +1,135 @@
+[paths]

```

Comment:
Having all these config files for the ner models is not super great, as presumably they are identical apart from the paths. Is there a way to generate these templates programmatically somehow so we can avoid this?

Reply:
I think we can use https://nightly.spacy.io/api/cli#init-fill-config to eliminate the boiler plate repetition, just haven't looked into it yet.

Reply:
used overrides and reduced to three distinct config files